speaking-portal-project/src/types.ts

```diff
@@ -15,7 +15,6 @@ export type Phoneme = 'A' | 'B' | 'C' | 'D' | 'E' | 'F' | 'G' | 'H' | 'X'
 export enum AvatarNames {
     barb,
     boy,
-    __LENGTH
```

Comment:
This is the only important change, the rest of the changes are related to code linting

Reply:
Great change and nice catch in peer testing